js/client/modules/@arangodb/testutils/process-utils.js

```diff
@@ -608,10 +621,14 @@ function runArangoImportCfg (config, options, rootDir, coreCheck = false) {
 // //////////////////////////////////////////////////////////////////////////////
 
 function runArangoDumpRestoreCfg (config, options, rootDir, coreCheck) {
+  config.calculateLogFile();
   if (options.extremeVerbosity === true) {
     config.print();
   }
-  return executeAndWait(config.getExe(), config.toArgv(), options, 'arangorestore', rootDir, coreCheck);
+
+  let ret = executeAndWait(config.getExe(), config.toArgv(), options, 'arangorestore', rootDir, coreCheck);
+  ret.message += config.getLogFile();
```

Comment:
This will attach the logfile content to the existing message without any separator/newline/delimiter.
Do we want to do something like
```
let logFileContents = config.getLogFile().toString();
if (logFileContents !== '') {
  ret.message += '\nContents of log file ' + this.config['log.file'] + ': ' + logFileContents;
}
```
to make it more readable?